.github/workflows/deploy_dev.yml

```
@@ -49,13 +49,50 @@ jobs:
+          gh pr create \
+            --title "Deploy FE: update image tag to ${{ needs.build-and-publish.outputs.image_tag }} and bump Chart version" \
+            --body "Automatically created PR to update helm manifests with the new image tag." \
+            --base main --head $BRANCH_NAME
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Merge Pull Request using gh
+        working-directory: cd
+        run: |
+          PR_NUMBER=$(gh pr list --head "deploy-dn-fe-dev-${{ github.run_id }}" --json number --jq '.[0].number')
+          if [ -z "$PR_NUMBER" ]; then  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)